### ISSUE TITLE
LinterPhp.Linter is deprecated.

### ISSUE THREAD

**Issue by Arcanemagus** [AtomLinter/linter-phpmd#9]
AtomLinter v0.X.Y API has been deprecated. Please refer to the Linter docs to update and the latest API: https://github.com/AtomLinter/Linter/wiki/Migrating-to-the-new-API
```
LinterPhpmd.Linter (C:\Users\Landon\.atom\packages\linter\lib\linter.coffee:60:4)
new LinterPhpmd (C:\Users\Landon\.atom\packages\linter-phpmd\lib\linter-phpmd.coffee:21:4)
Object.lint (C:\Users\Landon\.atom\packages\linter\lib\legacy.coffee:61:21)
<unknown> (C:\Users\Landon\.atom\packages\linter\lib\editor-linter.coffee:68:23)
<unknown> (C:\Users\Landon\.atom\packages\linter\lib\editor-linter.coffee:67:24)
Set.forEach (native)
```

**Comment by fredcamps** [comment]
+1

**Comment by steelbrain** [comment]
Fixed by #11
